reference/orders.v3.yml

```
@@ -1863,7 +1864,7 @@ components:
                 - paypalexpress
                 - paypalpaymentsprous
                 - paypalpaymentsprouk
-                - plugnpay
+                - plugnplay
```

AFAIK service is called `Plug'n Pay`, so I think this should remain `plugnpay`.

```suggestion
                - plugnpay
```